dvclive/live.py

```diff
@@ -46,7 +47,7 @@ def __init__(
 
         if self._report is not None:
             out = Path(self.html_path).resolve()
-            print(f"Report will be saved at {out}")
+            logger.info(f"Report will be saved at {out}")
```

Comment:
Can be a good time to address https://github.com/iterative/dvclive/pull/247#issuecomment-1116425371 . Any suggestions?

> Report path (if generated): 

^ Reads fine to me

Reply:
Yup, I think that's fine.